concepticondata/conceptlists/Morucci-2019-643.tsv

```diff
@@ -0,0 +1,644 @@
+Morucci-2019-643-571	571	stucchevole	sickly	2.686	1.657	0.943	1.486	2.886	2.886	vista	9.657	1.534	0.201	4.059	6.1	3.632		
+Morucci-2019-643-572	572	stupido	stupid	0.077	0.051	0.154	3.385	2.897	3.385	udito	6.564	0.881	0.508	6.789	2.471	6.368	1518	STUPID
+Morucci-2019-643-573	573	succoso	juicy	4.486	1.639	2.054	0.676	3.649	4.486	gusto	12.504	1.448	0.305	5.263	3.706	6.053		
+Morucci-2019-643-574	574	succulento	succulent	4.744	3.308	1.231	0.308	3.744	4.744	gusto	13.333	1.377	0.333	5.056	5.1	5.526		
+Morucci-2019-643-575	575	sudato	sweaty	1	4.359	4.231	0.385	4.564	4.564	vista	14.538	1.364	0.287	6.778	2.7	6.368		
+Morucci-2019-643-576	576	suggestivo	suggestive	1.65	1.975	1.15	3	4.45	4.45	vista	12.225	1.5	0.27	6.176	5.333	5.778		
+Morucci-2019-643-577	577	superficiale	superficial	0.189	0.135	1.649	1.865	3.378	3.378	vista	7.216	1.212	0.449	6.5	5.3	5.947		
+Morucci-2019-643-578	578	sussurrante	whispering	0.026	0.053	0.026	4.897	1	4.897	udito	6.002	0.553	0.812	3	4.588	4.158		
+Morucci-2019-643-579	579	tacito	unspoken	0.125	0.15	0.075	4.05	1.975	4.05	udito	6.375	0.869	0.624	3.684	5.529	4		
+Morucci-2019-643-580	580	taciturno	taciturn	0	0	0.162	4.676	2.973	4.676	udito	7.811	0.755	0.599	4.722	5.1	5.105		
+Morucci-2019-643-581	581	tanfoso	thunderous	1.079	4.921	0.282	0.263	1.513	4.921	olfatto	8.058	1.114	0.578	1.474	5.688	3.053		
+Morucci-2019-643-582	582	tenero	tender	2.275	0.575	4.225	1.525	3.525	4.225	tatto	12.125	1.446	0.301	6.579	3.059	6.737	1954	GENTLE
+Morucci-2019-643-583	583	tentacolare	sprawling	0.675	0.1	3.949	0.225	4.692	4.692	vista	9.641	1.037	0.476	2.833	5.75	3.632		
+Morucci-2019-643-584	584	tenue	soft	1.432	1.405	0.757	2.378	3.703	3.703	vista	9.676	1.475	0.304	4.611	5.25	4.737		
+Morucci-2019-643-585	585	teoretico	theoretical	0.194	0.2	0.161	2.129	0.935	2.129	udito	3.619	1.117	0.544	2.842	6.059	3.526		
+Morucci-2019-643-586	586	terroso	earthy	1.412	2.235	4	0.618	4.676	4.676	vista	12.941	1.421	0.314	3.412	4.235	3.444		
+Morucci-2019-643-587	587	tiepido	lukewarm	2.941	0.676	4.5	0.324	1.206	4.5	tatto	9.647	1.278	0.433	6.333	3.1	6.316		
+Morucci-2019-643-588	588	tintinnante	tinkling	0.05	0.05	1.075	4.821	2.175	4.821	udito	8.171	0.993	0.584	4.412	4.938	4.778		
+Morucci-2019-643-589	589	tinto	stained	0.135	0.568	0.351	0	4.865	4.865	vista	5.919	0.64	0.822	5.765	4.471	5.667		
+Morucci-2019-643-590	590	tipico	typical	2.375	2.15	1.8	2.3	3.15	3.15	vista	11.775	1.592	0.115	6.684	3.824	5.316		
+Morucci-2019-643-591	591	tonante	thunderbolt	0	0.171	0.314	4.857	1.743	4.857	udito	7.086	0.832	0.685	2.778	5.85	3.053	3760	THUNDERY
+Morucci-2019-643-592	592	tondo	round	0.378	0.108	4.486	0.432	4.892	4.892	vista	10.297	1.018	0.465	6.579	2.765	6.389		
+Morucci-2019-643-593	593	torbato	peat	2.452	2.613	0.613	0.29	2.71	2.71	vista	8.677	1.383	0.279	2.235	6.364	2.778		
+Morucci-2019-643-594	594	torbido	cloudy	0.744	1.051	1.103	0.526	4.59	4.59	vista	8.013	1.258	0.507	5.647	5.389	4.944	2341	CLOUDY
```

Comment:
This is more "turbid" than "cloudy" (even though cloudy is not really wrong), but it is an adjective and the Concepticon set seems to refer to a noun only